projects/components/src/navigation/navigation-list.component.scss

```diff
@@ -85,4 +85,6 @@
 
 .expanded {
   width: 220px;
+  overflow-x: hidden;
+  resize: horizontal;
```

Comment:
do we want dynamic resize on the left nav? Since the widths are known ahead of time, we should have already sized appropriately (and I'm curious how the app responds to this changing size)

Reply:
updated